commit cf13c2f6991fa3039906ca36ac31b3866de6cce9
Author: Sune Vuorela <sune@vuorela.dk>
Date:   Mon Apr 17 15:19:49 2023 +0200

    Bump c++ standard version. Poppler is about to use c++17 types in the api, so better be prepared
    
    (cherry picked from commit bb1d6c7c9fdcac99db33b5f42d768603d698400c)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c8a14a661..216532286 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -93,7 +93,7 @@ elseif(WIN32)
     set(ENV{QT_PLUGIN_PATH} "$ENV{QT_PLUGIN_PATH};${KDE_INSTALL_FULL_QTPLUGINDIR}")
 endif()
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
